lab-davidw/README.md

```diff
@@ -0,0 +1,8 @@
+
+
+
+### attributions:
+---
+- I got stuck on how to use fs.readFileProm, and a couple other puzzles in storage.js and took a look at how another student had accomplished it: https://github.com/broxsonl/lab-08-09-single-resource-api/blob/master/
```

Comment:
thank you for acknowledging other people's work. 
Flesh this readme out a little bit more when you have a better idea as to what your app does, and what is necessary to run it